### ISSUE TITLE
Create alternative LT5/LT6 combo.

### ISSUE THREAD

**Issue by AbbyANoble** [StevenClontz/cal1-tbil-2021-checkit#11]
Francesca's suggestion:

Consider the following rational function.
<img width="171" alt="Screen Shot 2021-12-15 at 10 30 36 AM" src="https://user-images.githubusercontent.com/85640990/146215530-40481442-f54d-49f2-8826-1a54dc147510.png">

`\[ f(x) = \frac{2{\left(x + 3\right)} {\left(x + 1\right)}^{2}}{{\left(x - 2\right)}^{2} {\left(x +3 \right)} }\]`

- Find the horizontal asymptote(s) of `\(f(x)\)`, if there are any. Express your findings using limit notation.
- Find the vertical asymptote(s) of `\(f(x)\)`, if there are any. Express your findings using limit notation.
- Find the hole(s) of `\(f(x)\)`, if there are any. Express your findings using limit notation.
- Find the vertical and horizontal intercepts of `\(f(x)\)`, if there are any.
- Illustrate your findings with a careful hand-drawn sketch of the graph of `\(f(x)\)`.

Can easily randomize with other rational functions, should be kept in factored form (I think) to make things easier, should try to have the factors relatively small so it can be hand-graphed easily

**Comment by AbbyANoble** [comment]
I feel the last two parts align better with AD7. (And I would suggest x and y intercepts as opposed to vertical and horizontal intercepts.)

**Comment by StevenClontz** [comment]
From a materials design standpoint, if two learning outcomes are assessed together then I think they should be represented as the same outcome/section in the text.

**Comment by fragandi** [comment]
I agree with your considerations (I would say the last bullet point is more aligned with LT1) and could definitely cut the last two bullet points. My worry is that they can recognize the features of the graph from the formula, but then cannot translate them into graphical information. For example, many of my students would find the hole, but then would not draw a hole on the graph (because calculators do not show holes). So I could see cutting the second-last bullet point and then rewriting the last one to be more about "Draw a sketch of the graph of f(x) which shows the limits that you found in 1-3"

**Comment by AbbyANoble** [comment]
I remember this being a discussion in the development. I presented these together previously as a "Limits Involving Infinity" outcome, but they are often separate sections in books. My reason for assessing them together was (a) I was getting too many LOs and this was a conceivable place to trim and (b) knowing which type applied to which asymptote/method and making that distinction was important too. 

Re: @fragandi -- I would lean more toward something where they are asked to sketch in the asymptotes and the hole (when they exist) and not the rest of the function.

**Comment by fragandi** [comment]
Makes sense! To be fair when I thought of the sketch of the function, I mostly thought of putting the asymptotes and the hole in a sketch... which kinda is like the sketching the whole function though because between the horizontal and vertical asymptotes you basically have the general shape. But if you have a better way to ask this in a way that makes it clear that we really just want to see that they can draw the asymptotes and the hole... I am all for it!

**Comment by fragandi** [comment]
I will try to create this as LT7 for now just so that I can play with it before we create the alt bank
